modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -65,6 +67,21 @@ public OrchestratorApi(OrchestratorService orchestratorService,
         this.registerService = registerService;
     }
 
+    /**
+     * Register new service using ocl model.
+     *
+     * @return response
+     */
+    @Tag(name = "Service Vendor",
+            description = "APIs to manage register services.")
+    @Operation(description = "Get category list.")
+    @GetMapping(value = "/register/categories",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public Response<List<Category>> listCategories() {
```

Comment:
hi @baixinsui we must discuss why we need to wrap actual response object additionally in `Response` object. It makes sense to return only the actual response object.

Reply:
I think the original idea is to make the sider menubar items be determined by the backend categories.

Reply:
But In my view, fixed menu items are also ok for us.

Reply:
All interface return the Response object, which makes the client  operation of data more consistent.

Reply:
If the enums of Category change, now we only do some change in backend, not both backend and front.

Reply:
we dont need all interfaces to return response object. Thats the benefit of micro services. Each operation can return its own response object. At the moment i dont see any benefit in wrapping all actual return objects in Response object.

Reply:
Is there a unified moudle in front client for calling API and processing return data not yet?  and we will not create a moudle with this function?

Reply:
the client knows which API method returns what type if we define it correctly in the open api